docs/src/pages/components/badges/UnstyledBadge.tsx

```diff
@@ -74,13 +72,15 @@ function BadgeContent() {
 
 export default function UnstyledBadge() {
   return (
-    <Stack spacing={4} direction="row">
```

Comment:
Why not keep the Stack? It's meant as a CSS helper. It might where the limit between `@material-ui/system` and `@material-ui/core` is.

Reply:
The goal was to eliminate any dependency on `@material-ui/core`. Once we have a dedicated doc for the unstyled components I think it would be strange that we have the `@material-ui/core` as a dependency here. Anyway, no strong opinion as as you mention it is a CSS helper, so I could revert this change.

Reply:
Interesting problem, not sure what would be best, but no objection to moving forward with the current approach.

- Stack break the system only dependency narrative 
- Stack is superior to Box in this case
- Stack is not meant to be part of the examples, but to support the demo ("demo" is the whole file, "example" is one instance of the demo".
- Stack is not available in the system, maybe Grid, Typography, Stack should be in the system, generic style helpers unrelated to Material Design.

Reply:
> Stack is not available in the system, maybe Grid, Typography, Stack should be in the system, generic style helpers unrelated to Material Design.

This makes sense 👍 will leave the comment open for reference, and will keep the demo without the `Stack` component for now.

Reply:
What changed now? I remember also trying to remove unrelated components from demos and I think I was shut down.

To be clear: I'm very much in favor of removing unrelated components like Stack or Box because they make the learning curve steeper.

Reply:
Oops, the conversation got moved to https://github.com/mui-org/material-ui/pull/26869#issuecomment-866307829.